common/display/displayqueue.h

```diff
@@ -341,7 +341,8 @@ class DisplayQueue {
   int state_ = kConfigurationChanged;
   PhysicalDisplay* display_ = NULL;
   SpinLock power_mode_lock_;
-  bool handle_display_initializations_ = true;  // to disable hwclock monitoring.
+  bool handle_display_initializations_ =
+      true;  // to disable hwclock monitoring.
```

Comment:
I really don't like how the script handled this - no reason to split the assignment. I'd suggest putting the comment on the line before or even think about renaming this var so we don't need the dumb comment.

Reply:
Agreed. I will put the comment on the previous line to make this patch change just reformatting and probably do another patch to look at how to do away with bloated comments that explain things in favor of better naming conventions.